readme.md

```diff
@@ -94,6 +118,34 @@ const Employee = db.rawTable('Employee');
 
 ### Methods
 
+Every method can override the retry [options](https://github.com/tim-kos/node-retry#retryoperationoptions) passed with the `.connect()` method or can customise the retry configuration for the specific method.
+
+```js
+Employee
+	.find({Organisation: 'Amazon'})
+	.where({Salary: {$gt: 3000}})
+	.select('FirstName Name')
+	.retry({retries: 2})
+	.exec()
+    .then(employees => {
+        // => [{FirstName: 'Foo', Name: 'Bar'}]
```

Comment:
Indentation

Reply:
Could you give an suggestion on what the identation needs to be? It has the same identation as other documentation (see line 154)

Reply:
Because of refined GitHub I noticed that it has spaces instead of tabs. Couldn't see the difference on the mobile app.

![image](https://user-images.githubusercontent.com/1913805/73597165-84127800-4529-11ea-8f44-4b61f746a7e5.png)